### ISSUE TITLE
Add linter to check for unnecessary trailing zeros

### ISSUE THREAD

**Issue by sds** [brigade/scss-lint#213]
Create a linter `TrailingZero` which checks for unnecessary trailing zeros. For example:

**Bad**

``` scss
p { margin: 1.50em; }
```

**Good**

``` scss
p { margin: 1.5em; }
```
